packages/web/src/app/components/memberApprovalRequiredToggle.tsx

```diff
@@ -45,6 +46,9 @@ export function MemberApprovalRequiredToggle({ memberApprovalRequired, onToggleC
         }
     }
 
+    const isDisabled = isLoading || forceMemberApprovalRequired !== undefined;
+    const showForceMessage = forceMemberApprovalRequired !== undefined;
```

Comment:
similar to the `forceEnableAnonymousAccess` flag, we probably only want to show this message if `forceMemberApprovalRequired` is true